scripts/export.py

```diff
@@ -125,6 +128,13 @@ def nemo_export(argv):
         in_args["max_dim"] = args.max_dim
         max_dim = args.max_dim
 
+    if args.cache_support:
+        export_cache_support_prev = model.encoder.export_cache_support
```

Comment:
Check if model.encoder has the argument, if not raise an error for invalid argument.

Reply:
done.

Reply:
Do we actually need this as the option? Why don't we always export with cache support ?

Reply:
Yes, as a streaming model may also get exported to be used as offline model.